client/src/components/LinkDisplay/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { useRef, useState } from 'react';
+import { FiLink, FiCopy } from 'react-icons/fi';
+import './style.css';
+
+const LinkDisplay = ({ content }) => {
+  const textA = useRef(null);
+  const [buttonText, setButtonText] = useState('Copy');
+
+  const copyCodeToClipboard = () => {
+    const el = textA;
```

Comment:
This variable assignment is redundant. Use the ref directly - `textA.current.select();`